templates/zerver/help/include/api-super-user-only.md

```diff
@@ -0,0 +1,3 @@
+!!! warn ""
+    This endpoint is only available to API super users.
+    [Contact us](/help/contact-support) if you need to have this enabled.
```

Comment:
We probably should link to the management command docs for `manage.py change_user_role` for self-hosted installations wanting this?

Reply:
Maybe also extend this text to be something like this?
"""
This endpoint is limited to organizations administrators who additionally have the `is_api_super_user` 
permission for the Zulip organization.  This special permission, which also allows using the `send_message` API
to send messages that appear to be from any user in the organization (intended for mirroring content from 
another chat tool).

Zulip cloud users can request the `is_api_super_user` permission for a bot by contacting [Zulip Cloud support](/help/contact-support) with an explanation for why you need this permission.

Self-hosted installations can toggle `is_api_super_user` on an account using `manage.py change_user_role`.
"""

Reply:
Makes sense, added this with minor wording tweaks